nixos/modules/programs/steam.nix

```diff
@@ -18,7 +25,7 @@ in {
 
     hardware.steam-hardware.enable = true;
 
-    environment.systemPackages = [ pkgs.steam ];
+    environment.systemPackages = [ steam steam.run ];
```

Comment:
All the changes to `programs.steam` are there to pass system graphics drivers down to Steam's expression, which in turn passes them to build-fhsuserenv-bubblewrap, adding them to the ld cache that will be created in the container. This is also the part I alluded to in

> find a better way to pass the system's drivers in a way that pressure-vessel will find them, allowing steam to be added and work without activating the module

Plus, `steam.run` was added as a convenience, since to add that manually the user would have to replicate the logic of the module.

Reply:
also, they only slightly diverge, the closure cost is essentially a small wrapping script